src/components/shared/basicGrid/styles/basicGrid.scss

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
The style sheet should be a sibling file to the  .js file.

Reply:
It wasn't working that way.   So I followed the pattern used in the other shared components.  Though, since you want me to rearrange all these files, I'll try it again.